cmd/kubeadm/app/phases/patchnode/patchnode.go

```diff
@@ -41,3 +40,20 @@ func annotateNodeWithCRISocket(n *v1.Node, criSocket string) {
 	}
 	n.ObjectMeta.Annotations[constants.AnnotationKubeadmCRISocket] = criSocket
 }
+
+func RemoveCRISocketAnnotation(client clientset.Interface, nodeName string) error {
+	klog.V(1).Infof("[patchnode] Removing the CRI Socket annotation from the Node API object %q if it exists\n", nodeName)
+
+	return apiclient.PatchNode(client, nodeName, func(n *v1.Node) {
```

Comment:
```suggestion
	if err := apiclient.PatchNode(client, nodeName, removeNodeCRISocketAnnotation); err != nil {
		return errors.Wrapf(err, "could not remove the CRI socket annotation from node %s", nodeName)
	}
	return nil
```

this is done to allow the same error for all places that call RemoveCRISocketAnnotation